js/index.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
+import Bliss from 'bliss'
+import {TabbedContent} from './TabbedContent.js'
+
+document.addEventListener('DOMContentLoaded', function (event) {
+  function searchToObject (search) {
```

Comment:
I think this code could be used elsewhere. Can you move this into our `js/util.js` module?